scripts/kubecf-upgrade.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+source scripts/include/setup.sh
+
+require_tools helm kubectl
+
+helm upgrade kubecf "${CHART}" \
+     --namespace "${KUBECF_NS}" --values "${VALUES}" "$@"
```

Comment:
Doesn't `kubecf-apply.sh` already do `helm upgrade --install`? Why do we need a new script?

Reply:
My intent was to move away from doing `--set` anywhere. But yes, this should be fixed in `{cf-operator,kubecf}-apply.sh` instead of replicating targets.
Reverted. See:
https://github.com/cloudfoundry-incubator/kubecf/pull/1611/commits/38af78b4963f5f70f3c842e59d3a26c18425c7db
https://github.com/cloudfoundry-incubator/kubecf/pull/1611/commits/706daa62fb6698798d1b82fee5af842d0fd90533